Dockerfile-test

```diff
@@ -0,0 +1,9 @@
+FROM node:12-alpine
+
+ADD src /opt
+ADD package.json yarn.lock tsconfig.json tsconfig.build.json nest-cli.json .nvmrc /opt/
+WORKDIR /opt
+RUN yarn install && yarn cache clean
```

Comment:
1. No need for `install`.
2. Since this is not for development, shouldn't we use `yarn --frozen-lockfile`?
3. What's the point of the `cache clean` after the installation?

Reply:
1&2. Will do
3. It's a small optimization; docker "commits" a filesystem layer after running each command, so cleaning the cache on the same command as doing the install avoids commiting the cache and reduces the image size a bit. Not really important for what we're using it tho, just came to mind while I was writing it :P

Reply:
Makes sense :D.